frontend/src/scenes/organization/CreateOrganizationModal.tsx

```diff
@@ -9,23 +9,28 @@ export function CreateOrganizationModal({
     setIsVisible,
 }: {
     isVisible: boolean
-    setIsVisible: Dispatch<SetStateAction<boolean>>
+    setIsVisible?: Dispatch<SetStateAction<boolean>>
 }): JSX.Element {
     const { createOrganization } = useActions(organizationLogic)
     const [errorMessage, setErrorMessage] = useState<string | null>(null)
     const inputRef = useRef<Input | null>(null)
 
-    const closeModal: () => void = useCallback(() => {
-        setErrorMessage(null)
-        setIsVisible(false)
-        if (inputRef.current) inputRef.current.setValue('')
-    }, [inputRef, setIsVisible])
+    const closeModal: () => void = useCallback(
+        !setIsVisible
+            ? () => {}
+            : () => {
+                  setErrorMessage(null)
+                  setIsVisible(false)
+                  if (inputRef.current) inputRef.current.setValue('')
+              },
+        [inputRef, setIsVisible]
+    )
 
     return (
         <Modal
             title="Creating an Organization"
             okText="Create Organization"
-            cancelText="Cancel"
+            cancelButtonProps={!setIsVisible ? { style: { display: 'none' } } : undefined}
```

Comment:
Double negations are hard to read.

`setIsVisible ? undefined : { style: { display: 'none' } }`